runtime/autoload/zip.vim

```diff
@@ -331,9 +337,24 @@ fun! zip#Extract()
    call s:Mess('Error', "***error*** (zip#Extract) Please specify a file, not a directory")
    return
   endif
+  if filereadable(fname)
+   call s:Mess('Error', "***error*** (zip#Extract) <" .. fname .."> already exists in directory, not overwriting!")
+   return
+  endif
+  let target = fname->substitute('\[', '[[]', 'g')
+  if &shell =~ 'cmd' && (has("win32") || has("win64"))
```

Comment:
(trivial) This is redundant. This can be:
```vim
  if &shell =~ 'cmd' && has("win32")
```

Reply:
I'll fix it up, thanks